snorkel-team/snorkel-tutorials
# [WIP] Ludwig Integration for Spam Tutorial

## Description of proposed changes
Integrates Ludwig section into Spam tutorial

## Related issue(s)

Fixes # (issue)

## Test plan

## Checklist
[WIP]: Tests in progress

To Dos:

* Change Ludwig loss metrics (e.g. MSE -> Classification Loss)
* Update visualization calls to suppress irrelevant plots/title more transparently
* Confirm if we want `ludwig` in spam `requirements.txt`

Need help on these? Just ask!

* [] I have read the **CONTRIBUTING** document.
* [] I have verified that my changes are covered by continuous integration.
* [] All new and existing tests passed.
